docs/features.rst

```diff
@@ -37,14 +45,16 @@ Truly Distributed
 * No central server at all. All infrastructure is based on IPFS.
 * Still, central architectures can be build with ``brig``.
 * Simple user identification and discovery.
-* You do not store data you don't want to.
+* You do not store data you don't want to store. Pin what you need,
+  fetch the rest from the network on request.
 
 FUSE filesystem
 ---------------
 
-* FUSE filesystem that makes all files seem like a normal directory.
+* FUSE filesystem that acts your data is in a local directory.
```

Comment:
> FUSE filesystem that acts your data is in a local directory.

This sounds strange in English. How about:

"FUSE filesystem mirrors your data to a directory."